UI/test/integration-tests/singleSegmentSpeedLimitSpec.js

```diff
@@ -120,7 +120,7 @@ define(['chai', 'testHelpers'], function(chai, testHelpers) {
         });
         expect(features.length).not.to.equal(0);
         _.each(features, function(feature) {
-          expect(feature.getProperties().value).to.equal(100);
+          expect(feature.getProperties().value.value).to.equal(100);
```

Comment:
Can this "value.value" be changed to something more explicit?

Reply:
Let's leave that "value.value" like it is because to switch that we will have a lot of impact on the rest of the code